release_notes/40-release-notes.adoc

```diff
@@ -0,0 +1,447 @@
+* REM in {product-title-managed-short} (Field Trial)
+| DEP
+| REM
+
+| `/v1/report` endpoint
+| GA
+| GA
+| DEP
+
+| `/v1/serviceaccounts` endpoint
+| GA
+| GA
+| DEP
+
+| `VulnerabilityReports` permission
+| GA
+| DEP
+| REM
+
+| `vulns` field of `storage.Node` object in response payload of `v`/nodes`
+a|
+* DEP in {product-title-short} 3.73
+* REM in {product-title-managed-short} (Field Trial)
+| DEP
+| REM
+|====
+
+[id="deprecated-features-374"]
+=== Deprecated features
+
+The following section provides additional information about deprecated features listed in the preceding table.
+
+- The `--offline-mode` flag for the `roxctl scanner generate` command is deprecated, as Scanner's default behavior is to fetch vulnerability updates from Central. The flag is planned for removal in the {product-title-short} 4.2.0 release.
+- The Network Graph version 1.0 is deprecated. Use the Network Graph version 2.0 for improved functionality and a better user experience.
+- The PDF export feature in the Vulnerability Management section of the {product-title-short} web portal is deprecated and is planned for removal in the {product-title-short} 4.2.0 release. Use the vulnerability reporting feature instead for more comprehensive CSV data.
+- Vulnerability scanning support for non-RPM (ASP).NET Core Runtime is deprecated and is planned for removal in the {product-title-short} 4.2.0 release.
+- All `/v1/report` APIs for creating and managing vulnerability reports are deprecated and will be replaced with new `/v2/report` APIs in a future release.
+
+[id="remove-kernel-rn40"]
+==== Remove kernel module as collection method
+
+Currently, secured clusters can specify three options of collection methods for runtime events: eBPF (selected by default), kernel module, or no collection. Kernel module as a collection method was deprecated in the {product-title-short} version 3.74 release and is planned for removal in the {product-title-short} version 4.1 release.
+
+[discrete]
+[id="actions-kernel-module-rn40"]
+===== Actions you must take
+Verify the collection method of your secured clusters. This value is set in the `collector.collectionMethod` parameter and is one of the following methods:
+
+- `EBPF`
+- `KERNEL_MODULE`
+- `NO_COLLECTION`
+
+If any of your secured clusters uses `KERNEL_MODULE` as a collection method, change it to `EBPF`.
+
+[id="removed-features-rn40"]
+== Removed features
+
+The following section provides additional information about removed features listed in the preceding table.
+
+[id="permissions-rn40"]
+=== Permissions and permission sets
+
+As announced in the Release Notes for {product-title-short} 3.73.0, some permissions for permission sets are now grouped for simplification. The deprecation process will remove and replace the deprecated permissions with the replacing permission in the following list. The access level granted to the replacing permission will be the lowest among all access levels of the replaced permissions.
```

Comment:
Just checking whether "some permissions for permissions sets" was intended, and wasn't supposed to be just "some permissions sets" or "some permissions and permissions sets".

Reply:
@rhybrillou - I think you worked on this ticket - can you confirm that "some permissions for permission sets" is correct? (text is from https://github.com/stackrox/stackrox/blob/master/CHANGELOG.md#removed-features-1)

Reply:
I'm not sure what the proper wording should be.
The point is that :
- permission `ComplianceRuns` does not exist anymore. In order to keep the same level of functionality, the permission `Compliance` should be used instead with the same access level as before.
- the same applies to `AllComments`, `Config`, `DebugLogs`, `NetworkGraphConfig`, `ProbeUpload`, `ScannerBundle`, `ScannerDefinitions`, `SensorUpgradeConfig` and `ServiceIdentity`. There, the permission `Administration` should be used instead.

Reply:
Thank you, I think the language is good now.